src/IAM/Application/Logout/LogoutHandler.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Application\Logout;
+
+final class LogoutHandler
+{
+    public function __construct(private readonly RefreshTokenRepository $store) {}
```

Comment:
```suggestion
    public function __construct(private readonly RefreshTokenRepository $repository) {}
```

@SergeyGrinenko, since you renamed it, it's no longer a "store," is it?